src/styles/_overrides.scss

```diff
@@ -159,3 +159,13 @@
   margin-left: 1.375rem;
   margin-top: 2.5rem;
 }
+
+//---------------------------------------
+// Data table
+//---------------------------------------
+
+.component-example__live--data-table {
```

Comment:
I don't think we need this or the TableContainer component anymore right?

Reply:
Ah I meant to delete this. But the TableContainer component is being used in the `ComponentStatus` @vpicone